ocaml/xapi-idl/network/network_interface.ml

```diff
@@ -294,8 +294,8 @@ let err =
       def= errors
     ; raiser=
         (fun e ->
-          log_backtrace () ;
           let exn = Network_error e in
+          log_backtrace exn ;
```

Comment:
Should this be here at all? In xenops_interface.ml it isn't:
```
  ; raiser=
      (fun e ->
        let exn = Xenopsd_error e in
        error "%s (%s)" (Printexc.to_string exn) __LOC__ ;
        raise exn
      )
```

Reply:
If exceptions are re-raised, do we need `log_backtrace`? We'll end up with loads of partial backtraces in the logs then? I think the backtrace should be logged only at the "final" exception handler.

Reply:
That is the desire according to backtraces.md too, I'll remove these in a separate commit.

Reply:
Removed. I also found a duplicate backtrace in http-svr, removed this in a separate commit (`response_internal_error` already logs the backtrace, no need to do it also in the caller).